api.yml

```diff
@@ -0,0 +1,1580 @@
+openapi: 3.0.0
```

Comment:
- Can we please also add other responses 401, 404, 500
- Can we please add  description with query parameters. Eg: Page number for pagination
- Shall we add country/city in chapter? wyt?

Reply:
great suggestions. adding them in new PR